scripts/systemd/nutshutdown.in

```diff
@@ -1,2 +1,16 @@
 #!/bin/sh
-@SBINDIR@/upsmon -K >/dev/null 2>&1 && @SBINDIR@/upsdrvctl shutdown
+
+if @SBINDIR@/upsmon -K >/dev/null 2>&1; then
+  wait_delay=`/bin/sed -ne 's#^ *POWEROFF_WAIT= *\(.*\)$#\1#p' @sysconfdir@/nut.conf`
```

Comment:
nitpicking, but prefer @CONFPATH@ over @sysconfdir@, otherwise you can end up with something not fully expanded like "${prefix}/etc/nut.conf" when not specifying --sysconfdir and using default flags.
I'll do some more testing over the afternoon if possible... and I'll take care of this CONFPATH when merging too ;)